src/components/Company/Industry/IndustrySelect.tsx

```diff
@@ -22,11 +22,18 @@ export function IndustrySelect({
 }: IndustrySelectProps) {
   const formMethods = useForm<IndustryFormFields>()
   const { handleSubmit, setValue } = formMethods
-  const [items, setItems] = useState<ComboBoxItem[]>([])
+  const [items, setItems] = useState<ComboBoxOption[]>([])
 
   useEffect(() => {
     const loadItems = async () => {
-      setItems((await loadAll()).map(({ title: name, code: id }) => ({ id, name })))
+      setItems(
+        (await loadAll()).map(({ title, code }) => ({
+          id: code,
+          name: title,
```

Comment:
Can these just be removed? Isn't it just label/value for the combobox options?

Reply:
Lol you're faster than me, should be updated now